### DISCUSSION TITLE
Assuming this integration with the HTTP proxy is still the better way of integrating with Tesla?

### DISCUSSION THREAD

**Discussion by rubin110** [alandtse/tesla#1050] [General]
With the last two major HA updates I've spotted that the official integration now supports the fleet API. I'm having a hard time understanding if the HTTP proxy is still needed with that setup. It does seem like there are a lack of sensors with the official integration that this custom integration does have.

Anyhow, I'm guessing this custom integration is still the right one to run, correct?

Thanks!

**Comment by skipishere** [comment]
Currently the official integration is read only (due to signing not yet being implemented) so yeah this is the one to use if you want to be able to control your car/battery setup

**Comment by rubin110** [comment]
Thanks!
